### DISCUSSION TITLE
청취후기

### DISCUSSION THREAD

**Discussion by rebase** [DaleJule/dalejule.com#2] [청취후기]
다음 에피소드가 기다려집니다~

**Comment by sehyunnoh** [comment]
> 다음 에피소드가 기다려집니다~

다음 에피소드는 `프로그래밍 언어`란 주제로 8월 1일에 업로드 될 예정입니다. 관심 갖아 주셔서 감사합니다.

**Comment by juhui-jeong** [comment]
팟캐스트 잘 듣고 있습니다! 주제와 내용 모두 재미있습니다. 😄
